cmd/load-runner/main.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/signal"
+	"regexp"
+	"time"
+)
+
+type config struct {
+	duration       string
+	parsedDuration time.Duration
+	namespace      string
+	loadName       string
+	chartFilesPath string
+}
+
+const (
+	defaultReleaseNamePrefix = "rudder-load"
+	defaultChartFilesPath    = "./artifacts/helm"
+)
+
+func main() {
+	cfg := parseFlags()
+	if err := validateInputs(cfg); err != nil {
+		fmt.Fprintf(os.Stderr, "Validation error: %v\n", err)
```

Comment:
Why not use the go-kit structured logger? It's free and nice :smile:

Reply:
done